### PR TITLE
Update author headers to Flutter Authors.

### PR DESCRIPTION
**Pull request by godofredoc**

### CODE REVIEW

**Comment by Hixie** [comment]
Can you add a test to make sure people always use the same header? There's a similar test in Flutter/flutter if that helps. (and one in engine too I think)

**Comment by godofredoc** [comment]
> Can you add a test to make sure people always use the same header? There's a similar test in Flutter/flutter if that helps. (and one in engine too I think)

Added the test to verify the license headers.

File: licenses/check_licenses.dart [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:core' hide print;
+import 'dart:io' hide exit;
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as path;
+
+import 'run_command.dart';
+import 'utils.dart';
```
Comment (Hixie) [inline licenses/check_licenses.dart:13]: i recommend inlining these files into this one, and removing as much of the code as possible, e.g. it seems runCommand isn't needed at least

**Comment by Hixie** [comment]
Is the dart analyzer running against this new code? (as in `flutter analyze`)

File: licenses/check_licenses.dart (hunk 1 above)
Comment (godofredoc) [reply licenses/check_licenses.dart]: Done, moved all the functions to check_licenses.dat and removed the utility files.

**Comment by godofredoc** [comment]
> Is the dart analyzer running against this new code? (as in `flutter analyze`)

It was not running but added license folder to the tests.yaml to run the analyze checks.

**Comment by godofredoc** [comment]
> sis is actually running -- I would t

You were 

> > Is the dart analyzer running against this new code? (as in `flutter analyze`)
> 
> It was not running but added license folder to the tests.yaml to run the analyze checks.

Actually you were correct I forgot to upload the yaml file with the change for analyze and test license folder.

**Pull request merged**